### PR TITLE
[Improvement][Datasource]refactor datasource parsing other parameter

### PR DESCRIPTION
**Pull request by jon-qj**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/AbstractDatasourceProcessor.java [hunk 1]
```diff
@@ -86,4 +91,59 @@ public String getDatasourceUniqueId(ConnectionParam connectionParam, DbType dbTy
         BaseConnectionParam baseConnectionParam = (BaseConnectionParam) connectionParam;
         return MessageFormat.format("{0}@{1}@{2}", dbType.getDescp(), baseConnectionParam.getUser(), baseConnectionParam.getJdbcUrl());
     }
+
+    /**
+     * transform other
+     *
+     * @param type type
+     * @param otherMap otherMap
+     */
```
Comment (kezhenxu94) [inline dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/AbstractDatasourceProcessor.java]: This kind of JavaDoc is meaningless, please remove them or write more meaningful JavaDoc

Comment (jon-qj) [reply dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/AbstractDatasourceProcessor.java]: Get it and I modify it.

**Comment by kezhenxu94** [comment]
This PR seems to be far away from the original purpose, it's adding new alert plugin, please separate the feature into another pull request so that we can review more easily

**Comment by jon-qj** [comment]
> This PR seems to be far away from the original purpose, it's adding new alert plugin, please separate the feature into another pull request so that we can review more easily

Get it!

**Pull request closed without merging**